cmd/srcd/cmd/prune_test.go

```diff
@@ -100,6 +104,69 @@ func (s *PruneTestSuite) TestRunningContainers() {
 	require.Equal(netNames(prevNets), netNames(nets))
 }
 
+type PruneWithoutImagesTestSuite struct {
+	PruneTestSuite
+}
+
+func (s *PruneWithoutImagesTestSuite) TestRunningContainers() {
+	s.testRunningContainers(s.Require(), false)
+}
+
+func (s *PruneWithoutImagesTestSuite) TestStoppedContainers() {
+	require := s.Require()
+
+	out, err := s.RunCommand(context.TODO(), "prune")
+	require.NoError(err, out.String())
+}
+
+type PruneWithImagesTestSuite struct {
```

Comment:
question: why do you need another suite for `TestRunningContainersWithImages`?

Reply:
Just to run it separately as last suite in the Makefile. Eventually we could have a single suite and then change the Makefile to run the specific test in the suite.

Reply:
sorry, could you explain more your idea? I didn't really get it. Why can't you run `PruneTestSuite/TestRunningContainersWithImages` in makefile?

Reply:
That's actually the same thing I said 😄. We can have a single suite and two different tests and run `PruneTestSuite/TestRunningContainersWithImages` in the Makefile 👍. Gonna do it.